test/cypress/integration/card/moto-mask-payment.spec.js

```diff
@@ -0,0 +1,232 @@
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: false,
+      motoMaskCardNumberInput: true,
+      motoMaskCardSecurityCodeInput: true
+    })
+
+  const createPaymentChargeStubsMotoMaskCardNumberOnly = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardNumberInput: true
+    })
+
+  const createPaymentChargeStubsMotoMaskSecurityCodeOnly = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardSecurityCodeInput: true
+    })
+
+  const createPaymentChargeStubsMotoMaskCardNumberAndSecurityCode = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardNumberInput: true,
+      motoMaskCardSecurityCodeInput: true
+    })
+
+  const checkCardDetailsStubs = cardPaymentStubs.checkCardDetailsStubs(chargeId)
+
+  const confirmPaymentDetailsStubs = cardPaymentStubs.confirmPaymentDetailsStubs(chargeId, validPayment)
+
+  beforeEach(() => {
+    // this test is for the full process, the session should be maintained
+    // as it would for an actual payment flow
+    Cypress.Cookies.preserveOnce('frontend_state')
+  })
+
+  describe('Secure card payment page - MOTO NOT enabled', () => {
+    it('Should setup the payment and load the page', () => {
+      cy.task('setupStubs', createPaymentChargeStubsNoMoto)
+      cy.visit(`/secure/${tokenId}`)
+
+      cy.location('pathname').should('eq', `/card_details/${chargeId}`)
+      cy.window().its('chargeId').should('eq', `${chargeId}`)
+    })
+
+    it('Card number & security code input elements should NOT mask digits', () => {
+      cy.get('#card-no').invoke('attr', 'type').should('eq', 'text')
+      cy.get('#card-no').invoke('attr', 'autocomplete').should('eq', 'cc-number')
+
+      cy.get('#cvc').invoke('attr', 'type').should('eq', 'text')
+      cy.get('#cvc').invoke('attr', 'autocomplete').should('eq', 'cc-csc')
+    })
+  })
+
+  describe('Secure card payment page - MOTO NOT enabled but masking set for both card number and security code', () => {
+    it('Should setup the payment and load the page', () => {
+      cy.task('setupStubs', createPaymentChargeStubsNoMotoMaskCardNumberAndSecurityCode)
+      cy.visit(`/secure/${tokenId}`)
+
+      cy.location('pathname').should('eq', `/card_details/${chargeId}`)
+      cy.window().its('chargeId').should('eq', `${chargeId}`)
+    })
+
+    it('Card number & security code input elements should NOT mask digits as it is not a MOTO service', () => {
+      cy.get('#card-no').invoke('attr', 'type').should('eq', 'text')
+      cy.get('#card-no').invoke('attr', 'autocomplete').should('eq', 'cc-number')
+
+      cy.get('#cvc').invoke('attr', 'type').should('eq', 'text')
+      cy.get('#cvc').invoke('attr', 'autocomplete').should('eq', 'cc-csc')
+    })
+  })
```

Comment:
New test - when `MOTO=false` but `maskCardNumber=true` and `maskSecurityCode=true`